### ISSUE TITLE
Legacy SDKs is gone

### ISSUE THREAD

**Issue by louwers** [GoldenKappa/notSABS#9]
![image](https://user-images.githubusercontent.com/649392/42418794-0ba13b8a-82a8-11e8-87df-25dece65ac4a.png)
Seems like the instructions need to be updated?

**Comment by bergerbse** [comment]
It is there. "Enterprise License Key (Development)".

**Comment by louwers** [comment]
Do you mean the first option?

**Comment by cammybird** [comment]
Thanks bergerbse.... With your help,I found it.... It is in the "SAMSUNG EDU SKD"   Works like a charm!!!!!

THANK YOU!     You are the man!

**Comment by malcalevak** [comment]
This needs to get added to the documentation! It was the missing key (no pun intended) that I was looking for!

Edit: I see now that the documentation was updated by @GoldenKappa today, but it's not completely correct (at a glance, I saw it still mentioned the regular KNOX SDK).

This should close out issues #2, #4, #7, and #8 as well, once it's completely correct.

**Comment by cammybird** [comment]
I know,  I've been beating my head against the wall for a few weeks trying to figure it out.  Ouch.
Seems to work great once you get it all together.

**Comment by louwers** [comment]
@bergerbse Comment is spot on. Keeping this open until the docs are updated. :+1:

**Comment by GoldenKappa** [comment]
For anyone else whose keys have expired: https://seap.samsung.com/license-keys/create#section-samsung-edu-sdk

**Comment by desrod** [comment]
Just a heads-up: These keys no longer work for Adhell, Adhell2, Adhell3, SABS and notSABS. You can't generate a key that will successfully activate any longer. Shame.

**Comment by bergerbse** [comment]
No? It probably just hast a different name.

**Comment by desrod** [comment]
Nope, they're actively blacklisting keys and unless your app uses an SKL key (which none of the blocking apps do), then you're out of luck. I just had my SABS blacklisted literally 10 minutes ago. I disabled it, tried to re-enable it and it no longer starts, because the key is no longer valid. It's been running on my device for the better part of 2 years, activated and working fine. Now I'm dead in the water.

**Comment by cammybird** [comment]
I myself had the same issue with notSABS about a month ago..  I really liked notSABS but no matter what I couldn't get it to recognize my key from SEAP.   SABS and notSABS is GONE!  Won't activate.   So it is useless.  What I did do was download the apk for Adhell3 and get the KLM key from SEAP. BE SURE TO USE THE KLM KEY ONLY!   I installed Adhell3, the KLM key works and the app works great  Just load all the files with the "outlawwed" domains you used with SABS and Violla!   
Load apk editor pro on your phone and then search and run this file on it.  ah3_v3.1.1.235_97c7d574.apk
I believe you can find both on Mediafire.com   Here's the link,  http://m.mediafire.com/folder/sb37c6gmhqgbn/AdHell_3
Good Luck.

**Comment by meltylang** [comment]
My "ELM Development/EDU SDK" key still work (Issue date: Aug 29, 2018).

**Comment by cammybird** [comment]
Sabs or notSabs wouldn't recognize mine so after about a week of frustration, I went with Adhell.  I like it better too.

**Comment by honestbleeps** [comment]
Which adhell, cammybird? Can you link? There's several versions and I'm not sure what's trustworthy.

**Comment by cammybird** [comment]
I like the adhell3.1.1
Follow the link.

**Comment by cammybird** [comment]
I like the adhell3.1.1
Follow the link. I've  previously shown

Be sure you are guarded.
